<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectorForm</class>
 <widget class="QDialog" name="SectorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="bandwidhtLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bandwidth [MHz]:</string>
   </property>
  </widget>
  <widget class="QLabel" name="verticalLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical:</string>
   </property>
  </widget>
  <widget class="QLabel" name="azimuthLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Azimuth [deegres]:</string>
   </property>
  </widget>
  <widget class="QLabel" name="bandLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Band [MHz]:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="azimuthSpinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="horizonstaLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontal:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="powerDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="antennaLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Antenna:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mimoBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mimoLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MIMO:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="HorizontalToolButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QComboBox" name="bandwidthBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="characteristicLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Characteristic [file.csv]:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="VerticalToolButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="powerLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Power [dBm]:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="tiltSpinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>46</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="tiltLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tilt [deegres]:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bandSpinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SectorForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SectorForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
